html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_MainDisplay.js

```diff
@@ -1197,6 +1210,24 @@ class B747_8_FMC_MainDisplay extends Boeing_FMC {
             this.setInOut(template[13][0]);
         }
         SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex);
+        // Apply formatting helper to title page, lines and labels
+        if (this._titleElement !== null) {
+            this._titleElement.innerHTML = this._formatCell(this._titleElement.innerHTML);
+        }
+        this._lineElements.forEach((row) => {
+            row.forEach((column) => {
+                if (column !== null) {
+                    column.innerHTML = this._formatCell(column.innerHTML);
+                }
+            });
+        });
+        this._labelElements.forEach((row) => {
+            row.forEach((column) => {
+                if (column !== null) {
+                    column.innerHTML = this._formatCell(column.innerHTML);
+                }
+            });
+        });
```

Comment:
would you be able to specify exactly what changes this introduces?

Reply:
> would you be able to specify exactly what changes this introduces?

It's allow replace short defined macro to correct property:

Defined macro:
`    _formatCell(str) {
        return str
            .replace(/{white}/g, "<span class='white'>")
            .replace(/{blue}/g, "<span class='blue'>")
            .replace(/{yellow}/g, "<span class='yellow'>")
            .replace(/{green}/g, "<span class='green'>")
            .replace(/{red}/g, "<span class='red'>")
            .replace(/{inop}/g, "<span class='inop'>")
            .replace(/{small}/g, "<span class='s-text'>")
            .replace(/{sp}/g, "&nbsp;")
            .replace(/{end}/g, "</span>");
    }` 
    
    Call for coloring:
    `displayCurrentPilotsOption = `{small}NONE{end}/{green}PILOT{end}/{small}COPILOT{end}/{small}BOTH{end}``